acquire-driver-common/src/simcams/simulated.camera.c

```diff
@@ -217,20 +218,50 @@ simulated_camera_streamer_thread(struct SimulatedCamera* self)
 {
     clock_init(&self->streamer.throttle);
 
+    size_t bytes_of_data = aligned_bytes_of_image(&self->im.shape);
+    uint8_t* data = (uint8_t*)malloc(bytes_of_data);
+    int64_t frame_id = self->im.frame_id;
+
+    const float exposure_time_ms = self->properties.exposure_time_us * 1e-3f;
+
     while (self->streamer.is_running) {
+        if (self->properties.input_triggers.frame_start.enable) {
+            ECHO(lock_acquire(&self->im.lock));
+            while (!self->software_trigger.triggered) {
+                ECHO(condition_variable_wait(
+                  &self->software_trigger.trigger_ready, &self->im.lock));
+            }
```

Comment:
When the `frame_start` trigger is disabled, this should release the trigger.

So maybe the predicate here should be 
```suggestion
            while (self->properties.input_triggers.frame_start.enable && !self->software_trigger.triggered) {
                ECHO(condition_variable_wait(
                  &self->software_trigger.trigger_ready, &self->im.lock));
            }
```

Right now, `trigger_ready` gets signaled when there's a change in `frame_start` on line 387.